client/components/themes-list/docs/example.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ThemesList from 'components/themes-list';
+
+const onScreenshotClickFn = function( theme, index ) {
+	console.log( `Theme ${ theme.id } at ${ index } clicked` );
+};
+
+const ThemesListExample = React.createClass( {
+	displayName: 'ThemesList',
```

Comment:
So I had originally changed this to `ThemesListExample`, but it does not work when you click the **Themes List** heading to view only that one app component. Just recalled and realized that my latest commit still has that issue. Let me look at some other examples to see how that has been handled.